spec/controllers/donations_controller_spec.rb

```diff
@@ -222,4 +222,19 @@
       expect(response).to be_redirect
     end
   end
+
+  context 'calculating total value of multiple donations' do
```

Comment:
Since we're not making any http or route calls I think this test would be better served as part of the donations model test.

Reply:
I put it in the controller spec because it is testing a method on the controller. However, I agree with you that it a more model-y method. Can we move `total_value` to somewhere in model code? If so, would it be appropriate to put it in the Donation model itself, even though it deals with multiple donations?

Reply:
Ah, I totally assumed it was in the model code already 😆. 

Now I'm wondering if `total_value` might belong in Itemizable itself. Hmm. @armahillo what are your thoughts? Times like these sure make me wish I had service objects and calculators here, lol

Reply:
Since it's modifiying an existing test let's leave it as-is.

I agree this is a model spec constraint though. Could move that in a different PR?